snap/snapcraft.yaml

```diff
@@ -310,6 +318,21 @@ parts:
         git describe --tags > ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/edge-proxy.VERSION
       organize:
         bin/edge-proxy: wigwag/system/bin/edge-proxy
+    pe-terminal:
+      plugin: go
+      source: https://github.com/PelionIoT/pe-terminal.git
+      source-commit: 8d2a020eb0bccf40cd9d9e54188e29df531bc019
```

Comment:
Our new preference is to use source-tag instead of source-commit.  Does your repo have a tag available instead of a sha?